packages/layers/src/xr-3d-layer/xr-3d-layer.js

```diff
@@ -324,7 +324,7 @@ const XR3DLayer = class extends Layer {
         { disableWanings: false }
       );
       model.setBindings(textures);
-      model.draw(opts);
+      model.draw(opts.renderPass);
```

Comment:
```suggestion
      model.draw(this.context.renderPass);
```

The layers in the DeckGL repository all use the `this.context.renderPass` pattern. Any particular reason to use the `opts.renderPass` instead?

https://github.com/visgl/deck.gl/blob/ade225112c3749651cd9aee590641b30d3aea001/modules/layers/src/bitmap-layer/bitmap-layer.ts#L291

Reply:
I've committed a change as suggested, it seems to work and tests pass.